content/en/tools/digital-currencies/transferring/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+date: '2023-06-23'
+lastmod: '2023-06-23'
+section_menu: tools
+menu:
+  tools:
+    name: "Working with Token Transfers"
+    weight: 1200
+    parent: digital-currencies
+    identifier: digital-currencies-tokens-transferring
+    description: "Digital Currencies documentation describing how to transfer tokens via the GUI"
+title: "Working with Token Transfers"
+---
+
+Once a participant has been [issued tokens]({{< relref "../issuing/_index.md" >}}), they can be involved in transfers. A {{< tooltip >}}transfer{{< definitiondc term="transfer" >}}{{< /tooltip >}} is the movement of tokens (value) between two or more entities (for example, wholesale banks, bank branches, retailers, retailer branches or franchises) on a Corda network to exchange goods and services. The transfer process can work both ways: both *push requests* and *pull requests* can be made.
```

Comment:
do we still need `definitiondc term="transfer"` ?

Reply:
Removed.